gojira.sh

```diff
@@ -560,6 +566,7 @@ function executable {
 function p_compose {
   local res
   get_envs
+  add_egg cli_envs
```

Comment:
Not 100% on having this on `p_compose`, since this function might be called multiple times, resulting on duplicated entries of `cli_envs`.

Reply:
Why not by the end of `parse_args`, and only to be run if `[[ -n $_GOJIRA_CI_ENVS ]]`?

If it's too much hassle working with the egg, we can just add these to the compose yml file directly.